app/lib/features/chat/widgets/create_chat.dart

```diff
@@ -124,14 +124,14 @@ class _CreateChatWidgetState extends ConsumerState<CreateChatPage> {
         config.addInvitee(userId);
       }
 
-      if (convoName != null && convoName.isNotEmpty) {
+      if (convoName != null) {
         // set the name
-        config.setName(convoName);
+        if (convoName.isNotEmpty) config.setName(convoName);
       }
 
-      if (description != null && description.isNotEmpty) {
+      if (description != null) {
         // and an optional description
-        config.setTopic(description);
+        if (description.isNotEmpty) config.setTopic(description);
```

Comment:
I don't understand these changes. It was clearer before without the double-if and proper blocks...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/86828671a4869455361bf9db46c158a6173842e5